library/json-safe.ecf

```diff
@@ -18,5 +18,6 @@
 			<cluster name="json_utility" location=".\utility\" recursive="true"/>
 		</cluster>
 		<cluster name="json_converter" location=".\converter\" recursive="true"/>
+		<cluster name="new_json_converter" location=".\new_converter\" recursive="true"/>
```

Comment:
I think this should or could be a separate library to keep the json library minimal and focus on parser+visitor+pretty printer.

This is what we decided months ago.
Maybe it could be inside the same eiffelhub/json project, or in a separated project.
I guess same project is better, but as a new library based on json-safe.ecf lib.